easybuild/easyconfigs/p/PyYAML/PyYAML-5.1.2-GCCcore-8.3.0-Python-3.7.4.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'PyYAML'
+version = '5.1.2'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
@Flamefire why not use Python `multi_deps` as is done with `PyYAML-5.1-GCCcore-8.2.0.eb`?

Reply:
Same as done before (some of my recent PRs): Python 2 will drop out of support by the end of the year, so I'd not add it to new ECs.

Reply:
Python 2 is still supported by PyYAML, and it will still work next year, so I don't see why we should not add it?
unless it was decided to move away from Python 2 software in EB that I don't know about. maybe @boegel can shed some light?

Reply:
See related discussion at https://github.com/easybuilders/easybuild-easyconfigs/pull/9251#issuecomment-552602776

Reply:
BTW: This does not exclude adding another EC for PyYAML with Python 2 if the need arises. But having to add it encourages not doing so and using Python 3. Especially: There are ECs for older Toolchains. If you use the newest toolchain, why not the newest Python?

Reply:
I'm also in favor of making this compatible with both Python 2 & 3 via `multi_deps`.

I understand your reasoning, and I mostly agree with it, but I don't consider making the `PyYAML` installation Python 2-compatible the same as encouraging the use of Python 2, especially since you'll get Python 3 by default when loading PyYAML (if Python 3 is listed first in `multi_deps`).

Correct me if I'm right, but you're trying to maintain a policy where you simply don't install a Python 2 module anymore?
How about hiding the Python 2 module instead, so it doesn't show up in `module avail`?
Then only people who know it's actually installed would consider loading it.

I consider that a better approach short-term (long-term I agree that we should stop providing Python 2, but we're not quite there yet imho, there's still stuff out there that really required Python 2, see for example #9176).

Hopefully we won't have this same discussion again and again in future PRs. ;)

Reply:
> Hopefully we won't have this same discussion again and again in future PRs. ;)

That's why I relied on previous PRs with the bottom line `allows to add another easyconfig for pkgconfig module with Python 2 if the need arises... `

Seems that doesn't apply to all ECs, so I'll make this Multi-Python again.

Reply:
That was a different situation, since the `h5py` for which `pkgconfig` was needed was stuck to Python 3 anyway because of `SciPy-bundle` being specific to Python 3.

That should be seen as an exception, not the rule. In general, we prefer installing with both Python 2 & 3 currently, since it leads to less easyconfigs to maintain, less modules installed, etc.

Reply:
Alright, updated